io.github.alexays.waybar.yaml

```diff
@@ -0,0 +1,235 @@
+      config-opts:
+        - -Dbuild-demos=false
+        - -Dbuild-tests=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz
+        sha256: 30d5bfe404571ce566a8e938c8bac17576420eb508f1e257837da63f14ad44ce
+
+  - name: chrono-date
+    buildsystem: cmake-ninja
+    build-options:
+      config-opts:
+        - -DBUILD_TZ_LIB=ON
+        - -DUSE_SYSTEM_TZ_DB=ON
+        - -DENABLE_DATE_TESTING=OFF
+        - -DCMAKE_BUILD_TYPE=None
+        - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/HowardHinnant/date/archive/v3.0.1.tar.gz
+        sha256: 7a390f200f0ccd207e8cff6757e04817c1a0aec3e327b006b7eb451c57ee3538
+
+  - name: fmt
+    buildsystem: cmake-ninja
+    build-options:
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=None
+        - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/fmtlib/fmt/archive/8.1.1.tar.gz
+        sha256: 3d794d3cf67633b34b2771eb9f073bde87e846e0d395d254df7b211ef1ec7346
+
+  - name: spdlog
+    buildsystem: cmake-ninja
+    build-options:
+      config-opts:
+        - -DSPDLOG_BUILD_BENCH=OFF
+        - -DSPDLOG_FMT_EXTERNAL=ON
+        - -DSPDLOG_BUILD_SHARED=ON
+        - -DCMAKE_BUILD_TYPE=None
+    sources:
+      - type: archive
+        url: https://github.com/gabime/spdlog/archive/v1.10.0.tar.gz
+        sha256: 697f91700237dbae2326b90469be32b876b2b44888302afbc7aceb68bcfe8224
+
+  - name: jsoncpp
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - --default-library=shared
+    sources:
+      - type: archive
+        url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.4/jsoncpp-1.9.4.tar.gz
+        sha256: e34a628a8142643b976c7233ef381457efad79468c67cb1ae0b83a33d7493999
+
+  - name: libmpdclient
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://www.musicpd.org/download/libmpdclient/2/libmpdclient-2.19.tar.xz
+        sha256: 158aad4c2278ab08e76a3f2b0166c99b39fae00ee17231bd225c5a36e977a189
+
+  - name: libnl
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/thom311/libnl/releases/download/libnl3_7_0/libnl-3.7.0.tar.gz
+        sha256: 9fe43ccbeeea72c653bdcf8c93332583135cda46a79507bfd0a483bb57f65939
+
+  - name: playerctl
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/altdesktop/playerctl/archive/v2.4.1.tar.gz
+        sha256: 75957ad5071956f563542c7557af16a57e40b4a7f66bc9b6373d022ec5eef548
+
+  - name: gtk-layer-shell
+    buildsystem: meson
+    build-options:
+      append-path: /usr/lib/sdk/vala/bin
+      env:
+        VALAC: /usr/lib/sdk/vala/bin/valac
+    sources:
+      - type: archive
+        url: https://github.com/wmww/gtk-layer-shell/archive/v0.9.0.tar.gz
+        sha256: 3809e5565d9ed02e44bb73787ff218523e8760fef65830afe60ea7322e22da1c
+
+  - name: intltool
```

Comment:
as I mentionned earlier, use the shared-modules for `intltool`.

Reply:
Sorry missed it.